src/components/swap/SwapHeader.tsx

```diff
@@ -13,14 +15,31 @@ const StyledSwapHeader = styled.div`
   color: ${({ theme }) => theme.textSecondary};
 `
 
+const TextHeader = styled.div<{ color: string; marginLeft?: string; isClickable?: boolean; disabled?: boolean }>`
+  color: ${({ color }) => color};
+  opacity: ${({ disabled }) => disabled && '60%'};
+  margin-left: ${({ marginLeft }) => marginLeft};
+  margin-right: 8px;
+  display: flex;
+  line-height: 20px;
+  flex-direction: row;
+  cursor: ${({ isClickable }) => isClickable && 'pointer'};
```

Comment:
i think you should be able to remove these new props now!

Reply:
ah youre right, done!